### DISCUSSION TITLE
Ctf error

### DISCUSSION THREAD

**Discussion by kajalssingh** [community/community#137593] [Mobile]
### Select Topic Area

Question

### Body

play store issue

**Comment by elisealix22** [comment]
Hi @Samrat0Suraj , I'm unsure what your error is with the Play Store, but here is a troubleshooting guide to fix common issues with downloading apps: https://support.google.com/googleplay/answer/14121800. I hope that helps!

**Comment by ebndev** [comment]
Hi @Samrat0Suraj, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share more information about your issue. Giving a few more details might help someone give you a nudge in the right direction. 😄
